src/main/resources/data/miningmaster/advancements/recipes/forging/ultima_pickaxe_1.json

```diff
@@ -30,25 +30,37 @@
         ]
       }
     },
-    "has_haste_peridot": {
+    "has_unbreaking_iolite": {
       "trigger": "minecraft:inventory_changed",
       "conditions": {
         "items": [
           {
             "items": [
-              "miningmaster:haste_peridot"
+              "miningmaster:unbreaking_iolite"
             ]
           }
         ]
       }
     },
-    "has_kinetic_opal": {
+    "has_haste_peridot_block": {
       "trigger": "minecraft:inventory_changed",
       "conditions": {
         "items": [
           {
             "items": [
-              "miningmaster:kinetic_opal"
+              "miningmaster:haste_peridot_block"
+            ]
+          }
+        ]
+      }
+    },
```

Comment:
Why is this now checking for a whole block of Haste Peridot? We may have talked about this, but I forget.